apps/settings/js/panels/call/call_settings_task_scheduler.js

```diff
@@ -0,0 +1,35 @@
+define(function(require) {
+  'use strict';
+
+  var CallSettingsTaskSchedulerConstant =
```

Comment:
Let's define the constant in this file following the comments on `TaskScheduler`.